src/main/java/de/themorpheus/edu/taskservice/database/repository/TaskDoneRepository.java

```diff
@@ -0,0 +1,17 @@
+package de.themorpheus.edu.taskservice.database.repository;
+
+import de.themorpheus.edu.taskservice.database.model.TaskDoneModel;
+import de.themorpheus.edu.taskservice.database.model.TaskModel;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.Optional;
+
+@Repository
+public interface TaskDoneRepository extends JpaRepository<TaskDoneModel, Integer> {
+
+	boolean existsByTaskId(TaskModel taskId);
+
+	Optional<TaskDoneModel> getTaskDoneByTaskId(int taskId);
```

Comment:
TaskModel

Reply:
Do you even start this and check if it runs properly?

Reply:
Yeah before I rebased

Reply:
> TaskModel

Why?? How should I get taskDoneId and Date?

Reply:
with your normal query but you need a foreign key